tests/unit/clients/redis_tests.py

```diff
@@ -117,6 +127,15 @@ def test_execute_command(self, monitored_redis_connection, expected_labels):
         )
         assert REGISTRY.get_sample_value(f"{REQUESTS_TOTAL._name}_total", expected_labels) == 1
 
+    def test_pipeline_instrumentation(self, monitored_redis_connection, expected_labels):
+        monitored_redis_connection.pipeline("test").set("hello", 42).set("goodbye", 23).execute()
+        labels = {**expected_labels, "redis_command": "SET", "redis_success": "true"}
+        assert REGISTRY.get_sample_value(f"{REQUESTS_TOTAL._name}_total", labels) == 2.0
+        assert (
```

Comment:
Please provide a descriptive error message. This isn't the current style, but it helps if your test fails in the future.

Reply:
Added